<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContextDlg</class>
 <widget class="QWidget" name="ContextDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KLineEdit" name="leConditionsFilter">
     <property name="inputMask">
      <string/>
     </property>
     <property name="clickMessage">
      <string>Filter...</string>
     </property>
     <property name="showClearButton" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="lvConditions"/>
   </item>
   <item>
    <widget class="QPushButton" name="pbNewCondition">
     <property name="text">
      <string>New Condition</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
